src/main/java/com/r307/arbitrader/config/AppConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.r307.arbitrader.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
+
+import java.util.concurrent.Executor;
+
+@Configuration
+public class AppConfig {
+
+    @Bean
+    public Executor taskExecutor() {
+        ThreadPoolTaskExecutor executor = new ThreadPoolTaskExecutor();
+        // Open to discussion
+        executor.setCorePoolSize(2);
```

Comment:
I believe setting this to 2 will start up two executors and allow two tasks to be executed concurrently. Let's set it to 1 and do some testing to be certain that we can't execute more than one copy of the trade logic at a time.

Reply:
Yes, you are right. It will create two threads which will allow two tasks to be executed concurrently.
I see we both agree in setting it to one :+1: